### PR TITLE
Adding ble support

### PR DESCRIPTION
**Pull request by amitbhanja**
1. Using pybluez for ble devices
   Tried it but it did not work as gattlib requires glib2.0 to be installed which was not possible in alpine container.
2. Bumble
    Tried bumble. It works by using the hci socket as the transport layer. However, it requires turning down the hci socket as bumble uses HCI_CHANNEL_USER while binding to the socket. This parameter gives exclusive access to the hci socket and cannot be used by another process. But sometimes, hci socket turns due to the bluetoothd service running and it creates conflicts. So, for resolving that we need to stop the bluetoothd.service which does not seem a viable solution for the user.

3. Therefore we have tried with bleak which uses d-bus and it does not require any other adjustments

### CODE REVIEW

File: nuvlaedge/peripherals/network/__main__.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 from nuvlaedge.peripherals.network import main
+import asyncio
 
-if __name__ == '__main__':
-    main()
+def entry():
```
Comment (ignacio-penas) [inline nuvlaedge/peripherals/network/__main__.py:4]: Could we keep the `if __name__ == '__main__':` check and entry() call in the __main__ file? That allows for the given package to be called from the cmd without the need of installing it into /usr/local/bin.

I also think the entry() function might fit better in __init__.py and then called from __main__ to keep the above mentioned functionality

Comment (amitbhanja) [reply nuvlaedge/peripherals/network/__main__.py:5]: Agree with your viewpoint. I have changed accordingly for all peripherals. Have tested on the nuvlaedge in rpi.

File: nuvlaedge/peripherals/gpu/__main__.py
Comment (schaubl) [inline nuvlaedge/peripherals/gpu/__main__.py:1]: There is a merge conflict that hasn't been resolved before commit here.

File: nuvlaedge/peripherals/network/__main__.py
Comment (schaubl) [inline nuvlaedge/peripherals/network/__main__.py:1]: There is also a merge conflict that hasn't been resolved before commit here.

**Review by ignacio-penas — APPROVED** [review]
All good from my side.

**Pull request merged**